doc/configuration.rst

```diff
@@ -2155,16 +2155,18 @@ If an ``int``, then that number of jobs will be passed to :class:`joblib.Paralle
 If ``True``, then the same number of jobs will be used as the ``-j`` flag for
 Sphinx.
 
-Warnings emitted by :mod:`joblib` during gallery generation (e.g., the ``UserWarning``
-about a `worker restarting <https://github.com/joblib/joblib/issues/883>`_),
-will be captured by Sphinx-Gallery in a similar way to warnings during example
-execution. This can be filtered out with ``warnings.filterwarnings``
-(see :ref:`removing_warnings`). This can be particularly important
-to do if you have tweaked warning handling in your doc build
+Python warnings emitted by :mod:`joblib` during gallery generation (e.g., the
+``UserWarning`` about a
+`worker restarting <https://github.com/joblib/joblib/issues/883>`_),
+will be captured by Sphinx-Gallery. These can be filtered out with
```

Comment:
Also not sure if these warnings are included in code execution output, so I've left at a vague 'capture'...?

Reply:
Actually thinking again, it may be to do with when the warning is emitted, at the time of gallery generation/example execution which is why it causes examples to fail (when warnings are error)

Reply:
I've amended the doc, hopefully it is right and makes sense